layouts/partials/head.html

```diff
@@ -24,6 +24,15 @@
   <!-- Additional local CSS -->
   <link rel="stylesheet" href="/css/style.css">
 
+  <!-- jQuery -->
+  <script src="https://code.jquery.com/jquery-3.5.1.js" integrity="sha256-QWo7LDvxbWT2tbbQ97B53yJnYU3WhH/C8ycbRAkjPDc=" crossorigin="anonymous"></script>
+  <!-- forms section -->
+  {{ if eq .Section "forms" }}
+  <!-- Select2 -->
+  <link href="https://cdn.jsdelivr.net/npm/select2@4.1.0-beta.1/dist/css/select2.min.css" rel="stylesheet" />
+  <script src="https://cdn.jsdelivr.net/npm/select2@4.1.0-beta.1/dist/js/select2.min.js"></script>
+  {{ end }}
+
   {{ if .IsHome }}
   <!-- DataTables -->
   <script src="https://code.jquery.com/jquery-3.5.1.js" integrity="sha256-QWo7LDvxbWT2tbbQ97B53yJnYU3WhH/C8ycbRAkjPDc=" crossorigin="anonymous"></script>
```

Comment:
without line 33 it is giving an error. I cannot find this JS above. Please point out where we have loaded this one.

Reply:
I meant of removing [line 38](https://github.com/ScilifelabDataCentre/covid-sample-collection-database/blob/FREYA-960/layouts/partials/head.html#L38) i.e. the `jquery` sourcing, since you are already sourcing it in line 28